modules/dnn/src/onnx/onnx_graph_simplifier.cpp

```diff
@@ -1704,29 +1704,63 @@ void simplifySubgraphs(opencv_onnx::GraphProto& net)
     simplifySubgraphs(Ptr<ImportGraphWrapper>(new ONNXGraphWrapper(net)), subgraphs);
 }
 
-Mat getMatFromTensor(const opencv_onnx::TensorProto& tensor_proto, bool uint8ToInt8)
+
+
+char* get_raw_data(const opencv_onnx::TensorProto& tensor_proto, const std::string base_path = ""){
```

Comment:
I'd suggest to modify this function (and probably rename it) to return `Mat` instead of plain pointer. This way we can track memory allocations properly. Currently, I suspect, there are memory leaks.